sources/advertisements/VASTMultipleCreatives.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
+<VAST version="3.0">
```

Comment:
This is a mix of multiple AdPods with multiple AdCreatives?

Reply:
Yes. In fact, I couldn't find an example where you have more than one ad pod, but this is just in case to check that we won't have any errors in parsing if we get a VAST like that.

Reply:
BTW maybe makes sense gather VPAID versions we receive and send them to telemetry? 
Looks like something - probably need to be discussed offline.